halo2_proofs/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ and this project adheres to Rust's notion of
 [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+## [0.3.0] - 2023-03-21
```

Comment:
I forgot to add `TracingFloorPlanner` to this changelog in #741 so we should do so here.